src/main/java/com/iota/iri/service/TipsSolidifier.java

```diff
@@ -36,7 +45,7 @@ public void init() {
                     scanTipsForSolidity();
                     if (log.isDebugEnabled()) {
                         long now = System.currentTimeMillis();
-                        if ((now - lastTime) > 10000L) {
+                        if ((now - lastTime) > getLogDelay()) {
```

Comment:
why not just extract to a static constant?

Reply:
Maybe one day we want to make this type of logging configurable.
+ magic values, they hurt my eyes!

Reply:
For those intervals we need to use the new `IntervalLogger` Hans introduced (not now of course).

Anyhows, some people are thinking that having too many configurations is bad. If we make it configurable, we will have to do `config.getLogDelay()`.

I think for just extract it to a constant. It makes the magic value leaner on the eye.